src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public  String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
+        String result = Colors.values()[index].toString();
```

Comment:
1. Please extract new Random() and Colors.values() to class variables to make code cleaner
2. No reason to introduce variable and instantly return it, just return expression you use for this variable